prescreeners/shared/css/custom-styles.css

```diff
@@ -75,7 +68,6 @@ fieldset.usa-fieldset legend {
 
 .label-sub-text {
   color: #2F4F4F;
-  font-size: 14px;
```

Comment:
Realized that this font size was smaller than the recommended USWDS default for accessibility; removing to make help text larger.